app/scripts/controllers/settings.js

```diff
@@ -87,6 +87,21 @@ angular.module('openshiftConsole')
           Logger.log("quotas", $scope.quotas);
         });
 
+        DataService.list("appliedclusterresourcequotas", context, function(quotas) {
+          $scope.clusterQuotas = quotas.by("metadata.name");
+          $scope.emptyMessageQuotas = "There are no resource quotas set on this project.";
```

Comment:
yeah im trying to decide what to do with it, because we only show the
message if both lists are empty

On Thu, Jul 7, 2016 at 3:36 PM, Sam Padgett <notifications@github.com>
wrote:

> In app/scripts/controllers/settings.js
> <https://github.com/openshift/origin-web-console/pull/140#discussion_r69972246>
> :
>
> > @@ -87,6 +87,21 @@ angular.module('openshiftConsole')
> >            Logger.log("quotas", $scope.quotas);
> >          });
> >
> > +        DataService.list("appliedclusterresourcequotas", context, function(quotas) {
> > +          $scope.clusterQuotas = quotas.by("metadata.name");
> > +          $scope.emptyMessageQuotas = "There are no resource quotas set on this project.";
>
> Hm, then it will switch to the message for both when one loads even if the
> other is still pending?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-web-console/pull/140/files/a99deea8356ff48171be55bc86e3fc0841280978#r69972246>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABZk7eCjC4SMyhpXVjREL1dQ0R0nh79Wks5qTVU1gaJpZM4JHZy8>
> .
>